src/docs/markdown/caddyfile/directives/handle_errors.md

```diff
@@ -49,5 +49,14 @@ Reverse proxy to a professional server that is highly qualified for handling HTT
 handle_errors {
 	rewrite * /{http.error.status_code}
 	reverse_proxy https://http.cat
+	header_up Host http.cat
```

Comment:
That's not exactly right - `header_up` is a subdirective of `reverse_proxy`, note the `{ }` in what I gave you. Also using the placeholder generalizes it, in that it uses the hostname from the proxy address (in this case it's always `http.cat` but if someone has more than one upstream, it can be dynamic) and avoids some slight duplication.

Reply:
There's a similar example in the `reverse_proxy` docs:

https://caddyserver.com/docs/caddyfile/directives/reverse_proxy#examples

Reply:
Oh, good catch. That's my bad

Reply:
Fixed it. Fought with git rebase to make it all pretty